tests/AutoReview/TravisTest.php

```diff
@@ -125,7 +138,7 @@ private function getAllPhpVersionsUsedByCiForTests()
         });
 
         return array_map(function ($job) {
-            return (string) $job['php'];
+            return \is_string($job['php']) ? $job['php'] : sprintf('%.1f', $job['php']);
```

Comment:
On PHP 8 it'll be `[7, 7.1, 7.2...]`, and casting first to float will result in just `7` not `7.0`.